addons/website_event_track/models/event_event.py

```diff
@@ -50,6 +55,43 @@ def _compute_sponsor_count(self):
         for event in self:
             event.sponsor_count = result.get(event.id, 0)
 
+    @api.depends('event_type_id', 'website_menu')
+    def _compute_website_track(self):
+        for event in self:
+            if event.event_type_id and event.event_type_id != event._origin.event_type_id:
```

Comment:
I think this is the only place where you check that we assign a **different** event_type_id.
Is there a specific reason?

Reply:
Becase of "website_menu" trigger. We should not copy template value if it has not changed. Worth a note I suppose.